nicsdru_nidirect_theme.theme

```diff
@@ -357,6 +357,29 @@ function nicsdru_nidirect_theme_preprocess_page_title(&$variables) {
       }
       break;
 
+    // Preprocess health conditions search page title.
+    case "view.health_conditions.search_page":
+      // Add a subtitle to indicate showing results and the current page selected from the pager.
+      $page = \Drupal::request()->get('page');
```

Comment:
I would assign \Drupal::request() to a local var and then call the subsequent ->get() statements from that.